packages/hooks/use-prop/index.ts

```diff
@@ -2,6 +2,6 @@ import { computed, getCurrentInstance } from 'vue'
 import type { ComputedRef } from 'vue'
 
 export const useProp = <T>(name: string): ComputedRef<T | undefined> => {
-  const vm = getCurrentInstance()!
-  return computed(() => (vm.proxy?.$props as any)[name] ?? undefined)
+  const vm = getCurrentInstance()
+  return computed(() => (vm?.proxy?.$props as any)[name])
```

Comment:
If vm is null, `(vm?.proxy?.$props as any)[name]` may throw an error.

Reply:
ok, chenge to `(vm?.proxy?.$props as any)?.[name]` will resolve the issue.

Test: 
```js
for (let n of [null, { proxy: null }, { proxy: { $props: null } }]) {
    vm = n
    name = 'temp'
    temp = vm?.proxy?.$props?.[name]
    console.log(temp)
}


```